parse_apartments.py

```diff
@@ -511,7 +511,9 @@ def main():
     """Read from the config file and get the Google maps info optionally"""
 
     conf = configparser.ConfigParser()
-    conf.read('config.ini')
+
+    # best to use the absolute path of the config file, prevents configparser.NoSectionError: No section: 'all'
+    conf.read('C:/whatever your path is/config.ini')
```

Comment:
The point of using a config file was to prevent people from touching the python code in any way, and this breaks that.
Also, the `C:/` assumes you're on a windows machine, and many people aren't (including me). I understand it's something that can be replaced, but it's confusing.
I assume `'./config.ini'` would ensure the lookup happens in the current folder.

Reply:
I have changed the way it is finding the config file to something more universal. The README was reverted.